src/coreclr/jit/compiler.h

```diff
@@ -10363,9 +10363,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     static fgWalkPreFn gsMarkPtrsAndAssignGroups; // Shadow param analysis tree-walk
     static fgWalkPreFn gsReplaceShadowParams;     // Shadow param replacement tree-walk
 
-#define DEFAULT_MAX_INLINE_SIZE 100 // Methods with >  DEFAULT_MAX_INLINE_SIZE IL bytes will never be inlined.
+#define DEFAULT_MAX_INLINE_SIZE 300 // Methods with >  DEFAULT_MAX_INLINE_SIZE IL bytes will never be inlined.
```

Comment:
I wonder if this (and other inlining variables) should be customizable via config knobs (assuming they aren't already).

It seems like the kind of thing that would be good to allow tweaking/customization of without recompiling so we can more easily get perf numbers or run stress tests.

Reply:
These already are, but in Debug only. I have an improvement in works to make this limit more dynamic and depend on:
1) Profile data
2) NonGenericCallsGeneric observation